pkg/api/v2/types.go

```diff
@@ -1353,3 +1353,29 @@ type CNIVersions struct {
 	// Versions represents the list of the CNI Plugin versions that are supported
 	Versions []string `json:"versions"`
 }
+
+// NetworkDefaults contains cluster network default settings.
+// swagger:model NetworkDefaults
+type NetworkDefaults struct {
```

Comment:
I wonder if we could use this one instead of new structs https://github.com/kubermatic/kubermatic/blob/189186b185bd40712370e79f18412b9a7887c9a9/pkg/apis/kubermatic/v1/cluster.go#L714

Reply:
Not a bad idea. I just wanted an API that is more convenient to use for the UI - e.g. explicit IPv4 / IPv6 defaults instead of parsing them form pod / services CIDR slices (where they can be mixed in any order).